hooks/mari/collector.py

```diff
@@ -246,6 +246,24 @@ def collect_texture_elements(self, settings, parent_item):
                     self.logger.info("Collected item: %s" % layer_item.name)
                     items.append(layer_item)
 
+                    # for each layer, add associated mipmap item
+                    item_name = "%s, %s (%s mipmap)" % (geo.name(), channel.name(), layer_name)
+                    mipmap_item = self._add_item(settings,
```

Comment:
Please also add `mari.mipmap` to `MARI_SESSION_ITEM_TYPES`

Reply:
Done.